### PR TITLE
P2022-57 Add Readme -> P2022-58 add README file

### PR DESCRIPTION
**Pull request by MarekMacko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Branch Template
+[SUB-TASK-NUMBER]-[SUB-TASK-NAME]
+`P2022-58-add-README-file`
+
+## Branch Commit
+[MAIN-TASK-NUMBER]-[MAIN-TASK-NAME] -> [SUB-TASK-NUMBER]-[SUB-TASK-NAME]
```
Comment (MarekMacko) [inline README.md:6]: Maybe we can drop a main ticket number?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Branch Template
+[SUB-TASK-NUMBER]-[SUB-TASK-NAME]
+`P2022-58-add-README-file`
```
Comment (MarekMacko) [inline README.md:3]: Good job!

File: README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## Branch Template
```
Comment (kmaslowiec) [inline README.md:1]: Some comment

File: README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+[SUB-TASK-NUMBER]-[SUB-TASK-NAME]
```
Comment (Nataniel-Antosik) [inline README.md:2]: Test comment

File: README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+`P2022-58-add-README-file`
+
+## Branch Commit
```
Comment (Nataniel-Antosik) [inline README.md:5]: Test comment

File: README.md (hunk 1 above)
Comment (Nataniel-Antosik) [reply README.md:6]: Test comment

**Review by Nataniel-Antosik — COMMENTED** [review]
Test comment

**Review by catchanus — COMMENTED** [review]
It's OK!

**Review by kmaslowiec — APPROVED** [review]
Nice :)

**Pull request merged**